plugins/kubernetes-backend/config.d.ts

```diff
@@ -54,6 +54,10 @@ export interface Config {
             skipTLSVerify?: boolean;
             /** @visibility frontend */
             skipMetricsLookup?: boolean;
+            /** @visibility secret  */
+            caData?: string;
+            /** @visibility secret  */
```

Comment:
Maybe didn't have to make the path as such be a secret? 🤔 But I'm fine either way, not sure that it's anybody's business to see this outside of the backend code that actually needs it

Reply:
I actually don't really understand how these comments work and was following my nose LOL. Is there a value for "only the backend needs it but it's not necessarily _secret_"?